test/src/locator.spec.ts

```diff
@@ -611,6 +611,76 @@ describe('Locator', function () {
         clock.restore();
       }
     });
+
+    it('click the last one successfully with multiple clickable elements', async () => {
+      const {page} = await getTestState();
+
+      await page.setViewport({width: 500, height: 500});
+      await page.setContent(`
+        <button id="test1" onclick="window.count++;">test1</button>
+        <div id="test2"></div>
+         <script>
+          setTimeout(() => {
+            const element = document.createElement("button");
+            element.id = "test3";
+            element.onclick = () => {
+              const div = document.getElementById("test2");
+              div.innerText = "test2";
+            };
+            document.body.append(element);
+          }, 50);
+        </script>
+      `);
+      await page.evaluate(() => {
+        // @ts-expect-error different context.
+        window.count = 0;
+      });
+      await Locator.last([
+        page.locator('#test1'),
+        page.locator('#test3'),
+      ]).click();
+      const count = await page.evaluate(() => {
+        // @ts-expect-error different context.
+        return globalThis.count;
+      });
+      const innerText = await page.evaluate(() => {
+        return document.getElementById('test2')?.innerText;
+      });
+      expect(count).toBe(0);
+      expect(innerText).toBe('test2');
+    });
+
+    it('click the last one successfully with only the last one is clikable', async () => {
+      const {page} = await getTestState();
+
+      await page.setViewport({width: 500, height: 500});
+      await page.setContent(`
+        <button id="test1">test1</button>
```

Comment:
button is always clickable, let's make test1 to be a div with width=0 and height=0 set via the style attribute?

Reply:
Tests have been modified. I still see the test failed. Can you help to check the failure? In my machine, the added tests can pass successfully.

Reply:
I am not able to reproduce on a Mac locally either

Reply:
seems to be Ubuntu-only